data/sen.csv

```diff
@@ -0,0 +1,42 @@
+col_name,breakdown _topic,filter_item,filter_grouping_column,filter_item_grouping,school_census_code
+sen_status,SEN status,Total,,,
+sen_status,SEN status,Any special educational need,,,
+sen_status,SEN status,No identified special educational need,,,
+sen_provision,SEN provision,"Education, health and care plan",sen_status,Any special educational needs,
```

Comment:
Is there a reason for having plural items for sen_status when grouping but in the first 3 rows we use singular versions of the same phrase? Would it be easier to pick one to help with matching data?

Reply:
Nope, that's a typo, good spot.